javascript/src/seedsource/containers/Map.jsx

```diff
@@ -372,21 +369,18 @@ class Map extends React.Component {
             }
         }
 
-        if (this.variableLayer !== null && this.variableLayer.options.opacity !== opacity) {
-            this.variableLayer.setOpacity(opacity)
-        }
-
-        if (this.resultsLayer !== null && this.resultsLayer.options.opacity !== opacity) {
-            this.resultsLayer.setOpacity(opacity)
+        if (this.displayedRasterLayers.length) {
```

Comment:
The `if` is unnecessary. If `this.displayedRasterLayers` is empty, then the `forEach` function, by definition, won't be called.